libethereum/WarpCapability.cpp

```diff
@@ -1,35 +1,27 @@
-/*
-    This file is part of cpp-ethereum.
-
-    cpp-ethereum is free software: you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation, either version 3 of the License, or
-    (at your option) any later version.
-
-    cpp-ethereum is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details.
-
-    You should have received a copy of the GNU General Public License
-    along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
-*/
+// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2019 Aleth Authors.
+// Licensed under the GNU General Public License, Version 3.
 
 #include "WarpCapability.h"
 #include "BlockChain.h"
 #include "SnapshotStorage.h"
+#include "libp2p/Common.h"
 
 #include <boost/fiber/all.hpp>
 #include <chrono>
 
+using namespace std;
```

Comment:
I actually prefer to avoid this. My opinion is that it improves readability to explicitly show what comes from the standard library.
Also @chfast complained that it can lead to hard-to-track compilation errors in cases of conflict between names of different namespaces.

Reply:
Ah okay, we do this in many source files so I did so here in the interest of consistency. What you're saying makes sense...I'll undo this change.